src/main/java/com/nekomaster1000/infernalexp/items/IWhipItem.java

```diff
@@ -1,13 +1,6 @@
 package com.nekomaster1000.infernalexp.items;
```

Comment:
Do we still actually need this interface? Could we possibly remove it as I don't think it really serves any purpose now

Reply:
That's a good point. I will commit that in a sec